bench/bench._test.go

```diff
@@ -0,0 +1,69 @@
+package bench
+
+import (
+	"encoding/binary"
+	"fmt"
+	"math/rand"
+	"os"
+	"reflect"
+	"testing"
+
+	"github.com/tetratelabs/wazero/wasi"
+	"github.com/tetratelabs/wazero/wasm"
+	"github.com/tetratelabs/wazero/wasm/naivevm"
+	"github.com/tetratelabs/wazero/wasm/wazeroir"
+)
+
+func BenchmarkEngines(b *testing.B) {
+	buf, _ := os.ReadFile("case/case.wasm")
```

Comment:
later we can think of a better name as it feels a placeholder otherwise for now (case/case.go|wasm)

Reply:
👍